tests/benchdnn/utils/task.hpp

```diff
@@ -43,8 +43,11 @@ struct task_t {
         , perf_template_(perf_template)
         , idx_(idx) {}
 
-    int create() {
-        BENCHDNN_PRINT(1, "create: %s\n", prb_.str());
+    int create(bool in_parallel) {
+        // Report creation status for problems only in sequanetial mode as in
+        // parallel it's still not clear which one failed.
```

Comment:
Sequanetial is mispelled.

Reply:
Thank you for spotting. Someday I manage vim to check spelling in comments...